### PR TITLE
Hide 0 plays on mobile just like desktop

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Trello Card Link
https://trello.com/c/F4sWZtX0/1643-0-play-counts-in-reposts-tab

### Description
Hides plays when 0 on mobile to replicate desktop behavior. The real bug is that get_repost_feed_for_user.py doesn't call populate track metadata in discprov, so it doesn't contain all the data we want.

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.
Dapp locally against prod

### CODE REVIEW

File: src/components/track/mobile/TrackTile.tsx [hunk 1]
```diff
@@ -42,6 +42,7 @@ type ExtraProps = {
 }
 
 const formatListenCount = (listenCount?: number) => {
+  if (!listenCount || listenCount === 0) return null
```
Comment (raymondjacobson) [inline src/components/track/mobile/TrackTile.tsx:45]: i guess this could just be !listenCount. updating

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/rj-hide-0-plays

**Review by piazzatron — APPROVED** [review]
LGTM! Any plans to fix the DP bug?

**Comment by raymondjacobson** [comment]
> LGTM! Any plans to fix the DP bug?

doing so now

**Pull request merged**